core/src/test/scala/com/salesforce/op/ModelInsightsTest.scala

```diff
@@ -456,6 +456,7 @@ class ModelInsightsTest extends FlatSpec with PassengerSparkFixtureTest with Dou
             i.featureName shouldEqual o.featureName
             i.featureType shouldEqual o.featureType
             i.derivedFeatures.zip(o.derivedFeatures).foreach { case (ii, io) => ii.corr shouldEqual io.corr }
+            o.distributions.foreach { o => o.cardEstimate shouldEqual None}
```

Comment:
the test could be more obvious asserting that the pre-serialization object has cardEstimate. And that it loses cardEstimates per requirement after serialization. As opposed to just checking the post-serialization state. 

However, I verified that the test fails without your main change, so we should be ok to merge it this way.

Reply:
@gerashegalov I updated the counter part of this test case